documentation/wallet.adoc

```diff
@@ -381,9 +448,64 @@ IMPORTANT: The unsealed private key is neither stored nor cached. It is utilized
 To sign a message, the content undergoes hashing using the Blake3 algorithm. The hashed value is subsequently signed using the ED25519 signature scheme. This function ensures the message signature process aligns with the Massa Labs cryptography standard.
 
 == Non-Functional Requirements
+=== Performance Requirements:
+** 1. Response Time:**
+The Massa Wallet module has a responsive user interface, with an
+ average response time for any user action (e.g., sending funds, 
+checking balances), not exceeding 2 seconds.
```

Comment:
I just put it for a reference. In fact, we are under 1 sec (don't know exactly the numbers) for all requests in MW.

Reply:
@Thykof here you can provide the correct number, not accounting for the latency when retrieving blockchain data, only the interface latency which should be in miliseconds (I suppose).

Reply:
We should rewrite this sentence because, sending funds and checking balances can take more that 1 second. What could be less that 1 second is displaying the user interface (http web page) but all the content inside (balances) can take longer to appear.

The display of the GUI depends on the performance of the computer of the user but I think we can say 800ms.